### DISCUSSION TITLE
how to hide menu title from admin sidbar?

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#643] [Q&A] [answered]
hi
i add category field to one admin view and automatically it show a menu in sidebar nav (as submenu) and in dashboard. is it possible to hide it?
![2021-01-04_12-23-51](https://user-images.githubusercontent.com/13805543/103518351-dae6a700-4e88-11eb-8766-2754ff357fb3.jpg)

**Comment by ohrionmartin** [comment, marked as answer]
Hi @saman222 

This [video](https://youtu.be/39vY66X7GGU?t=217) shows a good demonstration on how to achieve that, please watch it..

**Reply by saman222** [reply]
hi and thank you @ohrionmartin but i mean i want hide only category field. as you know i have an admin view that i added a category field to that. i want show that admin view but hide category field. as you can see in the picture, i have Room view and Room category and want show Room but hide Room category submenu. in that section of video that you get me , no description for Category field. thank you and wait for your advise

**Reply by ohrionmartin** [reply]
This functionality has now been added and to utilize it, you have to install the current staging branch of **JCB (_v2.12.5_)**, add a category field (Room(s) Category) to an Admin View (Room(s)). In the Fields' tab of your Admin View (Room(s)), you will find a switch that will allow you to select whether you would like a category to be added to the sub-menu.
 
![Screenshot from 2021-01-06 12-20-44](https://user-images.githubusercontent.com/38046346/103757635-b7069b00-5019-11eb-8421-b77b96058868.png)
